packages/nimble-components/src/anchor/tests/anchor.stories.ts

```diff
@@ -0,0 +1,68 @@
+import type { Meta, StoryObj } from '@storybook/html';
+import { withXD } from 'storybook-addon-xd-designs';
+import { html } from '@microsoft/fast-element';
+import { createUserSelectedThemeStory } from '../../utilities/tests/storybook';
+import '../../all-components';
+import { AnchorAppearance } from '../types';
+
+interface AnchorArgs {
+    label: string;
+    href: string;
+    underlineVisible: boolean;
+    appearance: keyof typeof AnchorAppearance;
+    disabled: boolean;
+}
+
+const metadata: Meta<AnchorArgs> = {
+    title: 'Anchor',
+    decorators: [withXD],
+    parameters: {
+        docs: {
+            description: {
+                component: ''
+            }
+        },
+        design: {
+            artboardUrl:
+                'https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/bfadf499-caf5-4ca0-9814-e777fbae0d46'
+        },
+        actions: {}
+    },
+    // prettier-ignore
+    render: createUserSelectedThemeStory(html`
+        Click on the <nimble-anchor
```

Comment:
Can you see if you can apply a nimble font to this text? Right now it looks like it's using the browser default Times New Roman for everything except the link.

Reply:
Done